cinema/views.py

```diff
@@ -118,21 +150,18 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(generics.ListCreateAPIView):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (OrderPermission,)
```

Comment:
The `OrderViewSet` uses `OrderPermission`, which should ensure that only authenticated users can create orders and only owners can observe them. Verify that the `OrderPermission` logic aligns with this requirement.